similarity/kNN_similar.py

```diff
@@ -63,9 +66,24 @@ def main():
   # K-Nearest Neighbors (k = 4)
   model = knn(vectorized)
   prediction = model.kneighbors(vectorized, 4, return_distance=False)
+  orgs = knn_pb2.Organizations()
   for i in range(df.shape[0]):
     print(df['Organization'][i], ': ', df['Organization'][prediction[i][1]], ', ',
         df['Organization'][prediction[i][2]], ', ', df['Organization'][prediction[i][3]])
+    org = orgs.orgs.add()
+    org.name = df['Organization'][i]
+    neighbor1 = org.neighbors.add()
+    neighbor1.name = df['Organization'][prediction[i][1]]
+    neighbor2 = org.neighbors.add()
+    neighbor2.name = df['Organization'][prediction[i][2]]
+    neighbor3 = org.neighbors.add()
+    neighbor3.name = df['Organization'][prediction[i][3]]
+  with open('neighbors.txt', 'wb') as out_file:
+    out_file.write(orgs.SerializeToString())
```

Comment:
Not necessary for this PR, but the main advantage of protos is their concise binary encoding.

This is how they get passed between different languages via RPCs.

This is just something to think about moving forward with Datastore / however you choose to store these values. Depending on the proto support in Datastore, you may need to restructure your proto definition.

Reply:
https://developers.google.com/protocol-buffers/docs/pythontutorial#parsing-and-serialization

I think SerializeToString() still returns a binary representation

Reply:
You're totally right, I just looked at the filename, saw that GitHub was able to display and stopped there, my bad.

Another common use of protos is storing them as human-readable text, which is what I thought this was doing.